test/tools/libtesteth/TestOutputHelper.cpp

```diff
@@ -100,3 +108,32 @@ void TestOutputHelper::printTestExecStats()
         saver.restore();
     }
 }
+
+void TestOutputHelper::registerTestFolders(boost::filesystem::path _path)
+{
+    // -t SuiteName/caseName   parse caseName as filter
+    std::string filter;
+    std::string const& tSuiteString = Options::get().rCurrentTestSuite;
+    std::size_t pos = tSuiteString.find('/');
+    if (pos != string::npos)
+        filter = tSuiteString.substr(pos + 1, tSuiteString.length() - pos);
+
+    using fsIterator = boost::filesystem::directory_iterator;
```

Comment:
Not sure if it's a good idea to browse the file system in a class responsible for helping with output, but fine.